wellcomeml/ml/cnn.py

```diff
@@ -98,9 +112,53 @@ def _get_distributed_strategy(self):
             strategy = tf.distribute.get_strategy()
         return strategy
 
+    def _init_from_data(self, X, Y):
+        logger.info(
+            "Initializing sequence_length, vocab_size \
+            and nb_outputs from data. This might take a while."
+        )
+        if type(X) is np.ndarray:
```

Comment:
`isinstance(X, np.ndarray)` will do too

Reply:
possibly better and more consistent, will dot the change

Reply:
b9bf570